src/mbgl/style/property_parsing.cpp

```diff
@@ -39,7 +39,21 @@ optional<std::string> parseConstant(const char* name, const JSValue& value) {
 
 template <>
 optional<Color> parseConstant(const char* name, const JSValue& value) {
```

Comment:
@jfirebaugh help : )

Defining

```
optional<std::array<float, 4>> parseConstant(const char* name, const JSValue& value) {
```

is considered a redefinition of `optional<Color>`. Is there a noninvasive way around this or do we just decide on the semantics here and maybe go with `optional<std::array<float, 4>>` as less misleading?

Other options?